apps/examples/testcase/ta_tc/systemio/itc/Kconfig

```diff
@@ -14,20 +14,79 @@ config SYSIO_ITC_UART
 	bool "Sysio UART TC"
 	default n
 
+if SYSIO_ITC_UART
+config SYSIO_ITC_UART_FILE_PATH
+	string "Sysio UART virtual test port"
+	default "/dev/ttyS1"
```

Comment:
Is there no way to get this value automatically from uart config?

Reply:
We don't have device port available in any UART config, so I think this should be ok